src/HDInsight/HDInsight/ManagementCommands/AddAzureHDInsightConfigValuesCommand.cs

```diff
@@ -22,9 +22,7 @@
 
 namespace Microsoft.Azure.Commands.HDInsight
 {
-    [GenericBreakingChange("Add-AzHDInsightConfigValues alias will be removed in an upcoming breaking change release", "2.0.0")]
     [Cmdlet("Add", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "HDInsightConfigValue"),OutputType(typeof(AzureHDInsightConfig))]
-    [Alias("Add-AzHDInsightConfigValues")]
```

Comment:
> So, to be clear, if you did not remove this alias at the last breaking chaneg release, you cannot remove it now. If you want to remove it at the next breaking change release, then leave the braking change notce and the alias here, and you can remove it in November.
> 
> Otherwise, please keep the alias and removing the braking change notice is OK.

Hi @markcowl  Thanks for your comments.  I decided to keep the alias and breaking change notice. I will remove it in November.

Reply:
> Note that removing this alias is one of the current static analysis failures preventing us from merging this PR.

As for static analysis failure, per my side I think it is false positive validation error.
Because the cmdlets in the breaking change result has been removed at the version 2.0.1.
And I have added suppression.

More details:
I exec `msbuild build.proj /t:StaticAnalysis /p:Configuration=Debug` at my local machine, I got the following breaking changes:

AssemblyFileName | ClassName | Target | Severity | ProblemId | Description | Remediation
-- | -- | -- | -- | -- | -- | --
Microsoft.Azure.PowerShell.Cmdlets.HDInsight.dll | Microsoft.Azure.Commands.HDInsight.GrantAzureHDInsightHttpServicesAccessCommand | Grant-AzHDInsightHttpServicesAccess | 0 | 1000 | The cmdlet 'Grant-AzHDInsightHttpServicesAccess' has been removed and no   alias was found for the original cmdlet name. | Add the cmdlet 'Grant-AzHDInsightHttpServicesAccess' back to the module,   or add an alias to the original cmdlet name.
Microsoft.Azure.PowerShell.Cmdlets.HDInsight.dll | Microsoft.Azure.Commands.HDInsight.RevokeAzureHDInsightHttpServicesAccessCommand | Revoke-AzHDInsightHttpServicesAccess | 0 | 1000 | The cmdlet 'Revoke-AzHDInsightHttpServicesAccess' has been removed and no   alias was found for the original cmdlet name. | Add the cmdlet 'Revoke-AzHDInsightHttpServicesAccess' back to the module,   or add an alias to the original cmdlet name.

I don't remove this two cmdlet in this PR, they have been removed at the version 2.0.1.